### ISSUE TITLE
v1.0 planning

### ISSUE THREAD

**Issue by AkihiroSuda** [lima-vm/lima#2771]
I'm planning to release Lima v1.0 right after the release of containerd v2.0 and nerdctl v2.0
- https://github.com/lima-vm/lima/pull/2178
- https://github.com/containerd/nerdctl/issues/2734
- https://github.com/orgs/containerd/projects/9

However, if containerd v2.0 and nerdctl v2.0 are not released by by November 8, I'm likely to release Lima v1.0 without containerd v2.0 and nerdctl v2.0.

**Comment by AkihiroSuda** [comment]
Beta 0: https://github.com/lima-vm/lima/releases/tag/v1.0.0-beta.0

**Comment by afbjorklund** [comment]
Do we want to add CID (i.e. for IPFS), for the images (ubuntu) and archives (nerdctl-full) that are included in v1.0?

The main downside is that you need some special tools, in order to update images (in addition to a SHA256 tool)

**Comment by AkihiroSuda** [comment]
> IPFS

Likely to be v1.1 or maybe v1.2, as we still haven't reached the consensus on the design?

**Comment by afbjorklund** [comment]
As long as we are going to use dedicated URLs (and not CID), we could use a gateway...

That is, if you want to use IPFS you can use https://{gateway URL}/ipfs/{CID}/{resource path}

Similar to how we can use `localhost:5050/ipfs/<CID>` with nerdctl? But 8080 instead.

```
http://localhost:8080/ipfs/QmUXmNgeYdjEt5RrcUaBfp3rGqeqwPWoHAR6UofU1AuYgu/ubuntu-24.04-server-cloudimg-amd64.img
```

**Comment by AkihiroSuda** [comment]
Released v1.0.0 🎉 
https://github.com/lima-vm/lima/releases/tag/v1.0.0

Will be available on Homebrew soon:
- https://github.com/Homebrew/homebrew-core/pull/196789
